tartiflette/engine.py

```diff
@@ -6,14 +6,7 @@
 
 
 class Engine:
-    def __init__(
-        self,
-        sdls,
-        schema_name="default",
-        _resolver_middlewares=None,
-        _resolvers=None,
-        _directive_resolvers=None,
-    ):
+    def __init__(self, sdls, schema_name="default"):
```

Comment:
Could you change `sdls` by `sdl`? The change is small but we have only one SDL.

Reply:
Or it could be a list of file names so technically it is "more than" one .sdl, but yeah, it forms only one entity in the end.
You sure I change it?

Reply:
Yes I'm sure

Reply:
Sure sure ? ;)

Reply:
Hum ... Yes :D